### DISCUSSION TITLE
Upgrading to 2.2.5 breaks my django app on windows

### DISCUSSION THREAD

**Discussion by spapas** [PyMySQL/mysqlclient#730] [Issue report]
### Describe the bug

When I upgrade to latest version (2.2.5) I'm getting the following error:

`django.db.utils.OperationalError: (2026, 'TLS/SSL error: SSL is required, but the server does not support it')`

Haven't change anything else in my app.

When downgrading to 2.2.4 everything works fine. 

Breaking changes cannot be introduced on minor versions.

### Environment

Windows 11, Python 3.11, Server version: 10.11.9-MariaDB MariaDB Server

### How did you install libmysqlclient libraries?

Using windows 11 and instaled the wheel with pip install.

### What version of mysqlclient do you use?

2.2.5

### Docker command to start MySQL server

```bash
Not needed
```

### Minimum but complete code to reproduce

```python
Not needed
```

### Schema and initial data required to reproduce.

```sql
Not needed
```

### Commands, and any other step required to reproduce your issue.

Not needed

**Comment by genesem** [comment]
Same thing. 
as a quickfix I did:

pip3 install mysqlclient==2.2.4

**Reply by spapas** [reply]
Yes I also did that but I'd prefer to be able to upgrade to the latest version :)

**Comment by methane** [comment]
duplicate of #729.
